src/core/Akka/Actor/ActorCell.DefaultMessages.cs

```diff
@@ -132,27 +132,38 @@ protected internal virtual void AutoReceiveMessage(Envelope envelope)
 
             switch (message)
             {
+                case ActorSelectionMessage selectionMessage:
```

Comment:
Re-organized the `switch` order for performance reasons - with the most frequent messages appearing first for short-circuiting reasons.